src/xdocs/index.xml.vm

```diff
@@ -217,6 +217,27 @@
       </div>
     </section>
 
+    <section name="Supported Language Level">
```

Comment:
Why Level?

It is first time I see this term for syntax version.

Reply:
I am familiar with this term from the IDEA project structure settings. see the message you received [here](https://github.com/checkstyle/checkstyle/pull/15480#issuecomment-2281952480). Also I took it from issue title and description

Reply:
`Supported Language features` ?

Reply:
done